settings.gradle.kts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+rootProject.name = "knotx-root"
+include(":knotx-core")
+include(":documentation")
+
+project(":knotx-core").projectDir = file("$rootDir/knotx-core")
```

Comment:
Redundant lines, project path is matching directory names

Reply:
fixed